src/news-update/index.md

```diff
@@ -0,0 +1,21 @@
+# What is a news update?
+
+News updates are a great way to garner your game some new or renewed attention and they are the primary means of staying in contact with your game's followers. Post news updates whenever something interesting happens with your game. Followers will be sent notifications and the news will be archived on your game page. Your news updates might even show up on the Game Jolt homepage.
+
+The best news updates deliver information that a game's followers would want to know. If you upload a new build of your game, you should take the time to make a news update. If your game just won a jam or an award, post a news update about it. If someone published a glowing review of your game, go ahead and make a news update.
+
+If you have something to post that is not a news update about a particular game, you can use your developer blog.
+
+## How do I post a news update?
+
+News updates trigger follower notifications and are an essential part of publicizing your game. To create a news update for a particular game, go to your [developer dashboard](http://gamejolt.com/dashboard/) and select the game from under "Your Games". On the game's overview page, visit the "News" tab and click "Add Game News".
+
+Now you can give your news update a title and write the content. The content is written in Markdown, which is easy to use and allows you to do a number of cool things without having to use HTML. For help with Markdown, consult our [Markdown guide](https://help.gamejolt.com/markdown).
+
+Be sure to space out your news updates. We want to offer a range of game news, so multiple posts made in a brief period of time for the same game will not appear on the homepage.
```

Comment:
How do I embed a video/image? you can't embed a video but you can do an animated gif via markdown